modules/rostests/apitests/ws2_32/recv.c

```diff
@@ -113,6 +113,21 @@ int Test_recv()
     ok(readIosb.Information == RECV_BUF, "Short read\n");
 
     NtClose(readEvent);
+
+    /* Now keep reading until the end (CORE-17425) */
+    int ret;
+    char recv_data[100] = { 0 };
```

Comment:
You could reuse one of these `szBuf1` buffers already declared in the function.
This would also allow the test to continue compiling in my setup and me not asking to move variables at the beginning of the function.